source/core/slang-char-encode.cpp

```diff
@@ -92,17 +92,17 @@ class Utf16CharEncoding : public CharEncoding //UTF16
 		Index index = 0;
 		while (index < length)
 		{
-			const Char32 codePoint = getUnicodePointFromUTF16([&]() -> Byte
-			{
-                if (index < length)
-                    return bytes[index++];
-                else
-                    return Byte(0);
-			});
+			auto readByte = [&]() -> Byte
```

Comment:
#Indentation seems off.

Reply:
Oops, thanks. This file is generally using tabs, and the pre-existing lines had a mixture of spaces and tabs.  I intended to have the newly added code be only tabs to match the local (though not Slang coding style compliant) style, but I missed a line or two.  The next patch revision will be all tabs in this spot.